pylint/checkers/refactoring.py

```diff
@@ -369,6 +372,48 @@ def visit_boolop(self, node):
                              node=node,
                              args=(duplicated_name, ', '.join(names)))
 
+    @utils.check_messages('consider-using-ternary')
+    def visit_assign(self, node):
+        if self._is_and_or_ternary(node.value):
+            cond, truth_value, false_value = self._get_and_or_ternary_arguments(node.value)
+        elif self._is_seq_based_ternary(node.value):
+            cond, truth_value, false_value = self._get_seq_based_ternary_params(node.value)
+        else:
+            return
+        self.add_message('consider-using-ternary', node=node,
+                         args=(truth_value.as_string(), cond.as_string(), false_value.as_string()),)
+
+    visit_return = visit_assign
+
+    @staticmethod
+    def _is_and_or_ternary(node):
+        """Returns true if node is 'condition and true_value else false_value' form.
+        All of: condition, true_value and false_value should not be a complex boolean expression"""
```

Comment:
Lets put the quotes on the next line and let's add a space between the these two sentences.